src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -2224,4 +2224,14 @@ public boolean isComplete() {
         return isComplete;
     }
 
+    public StringBuilder getCsvDetailedFeedbackResponseCommentsRow(FeedbackResponseAttributes response) {
+        List<FeedbackResponseCommentAttributes> frcList = this.responseComments.get(response.getId());
+        StringBuilder commentRow = new StringBuilder(200);
+        for (FeedbackResponseCommentAttributes frc : frcList) {
+            commentRow.append("," + frc.giverEmail + ","
+                    + SanitizationHelper.desanitizeFromHtml(
+                            frc.commentText.getValue().substring(3, frc.commentText.getValue().length() - 4)));
```

Comment:
Maybe extract this line out into a separate method to show this is intended to remove the `<p>` and `</p>` (if I'm reading this correctly.